exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/sklearn/sklearn_train_abalone.ipynb

```diff
@@ -52,14 +52,13 @@
    "metadata": {},
```

Comment:
<p>The split actually happened in the data notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Not sure I understood. Is this comment actionable?

Reply:
I see, it is for train and validation and train and test.